docs/reference/api/platform/Error.md

```diff
@@ -1,5 +1,20 @@
 ## Error
 
-The error function generates a fatal runtime error, which allows the user to dump a message to the serial line, halt the program and place the device in an idle mode with a blinking red LED.
+Mbed-OS provides an error function to output messages to STDIO at runtime when the system encounters a fatal error. Its declared in mbed_error.h. The function will output the error message when called for the first time, but will ignore subsequent calls without generating any output. 
```

Comment:
Might be worth adding that this function invokes exit - termination process . 

Sidequestion: I do not remember why are subsequent calls ingored?

Reply:
Also conditionally compiled with NDEBUG macro

Reply:
That's good idea, to add the termination/NDEBUG note.

Currently error function internally uses a static variable which prevents further output if you call again.